cyto_dl/nn/vits/mae.py

```diff
@@ -290,5 +295,7 @@ def __init__(
 
     def forward(self, img):
         features, mask, forward_indexes, backward_indexes = self.encoder(img, self.mask_ratio)
+        if self.features_only:
```

Comment:
if features_only mask_ratio must be 0?

Reply:
I don't want to enforce that in case we want to pass in a mask and just look at features from that subset of tokens (wishful thinking for now, but that's the reason behind that choice)